router/admin.go

```diff
@@ -59,6 +60,6 @@ func AdminHandlers(base *gin.RouterGroup) {
 		_admin.PUT("/user", admin.UpdateUser)
 
 		// Admin worker endpoint
-		_admin.POST("/workers/:worker/register-token", admin.RegisterToken)
+		_admin.POST("/workers/:worker/register", admin.WorkerRegistration)
```

Comment:
I like the change, but this will have downstream impact on the SDK. Just marking it here so we don't forget.